src/missed_spans.rs

```diff
@@ -20,15 +20,15 @@ impl<'a> FmtVisitor<'a> {
         self.format_missing_inner(end, |this, last_snippet, _| this.buffer.push_str(last_snippet))
     }
 
-    pub fn format_missing_with_indent(&mut self, end: BytePos) {
+    pub fn format_missing_with_indent(&mut self, end: BytePos, config: &Config) {
```

Comment:
This function should already have access to the configuration through `self.config`.